CMake/fletch-tarballs.cmake

```diff
@@ -544,6 +544,14 @@ set(YAMLcpp_md5 "e2507c3645fc2bec29ba9a1838fb3951")
 set(YAMLcpp_dlname "yaml-cpp-release-${YAMLcpp_version}.tar.gz")
 list(APPEND fletch_external_sources YAMLcpp)
 
+# qtExtensions
+set(qtExtensions_version "20180510git5c3772aa")
+set(qtExtensions_tag "5c37722aabf10e3d06262d6faf24991dad31626e")
+set(qtExtensions_url "https://github.com/Kitware/qtextensions/archive/${qtExtensions_tag}.zip")
+set(qtExtensions_md5 "bc2707dea007ad297e011406fac061bc")
+set(qtExtensions_dlname "qtExtensions-${qtExtensions_version}.zip")
```

Comment:
Thanks for using the dlname convention. To make it effective, you need to add the tag to the ExternalProjectAdd. DOWNLOAD_NAME${qtExtensions_dlname}. Otherwise, it looks good and can merge pending dashboards passing. I will run them after the requested patch.